quinn-proto/src/lib.rs

```diff
@@ -59,7 +59,10 @@ pub use crate::transport_error::{Code as TransportErrorCode, Error as TransportE
 
 pub mod congestion;
 
+mod cid_generator;
 mod token;
+pub use crate::cid_generator::ConnectionIdGenerator;
```

Comment:
Nit: Might swap this with line 63 and have an empty line after it for consistent style